move-to-sdk/app/io/sphere/sdk/facets/BaseFacetUI.java

```diff
@@ -0,0 +1,23 @@
+package io.sphere.sdk.facets;
+
+import io.sphere.sdk.models.Base;
+
+abstract class BaseFacetUI extends Base implements FacetUI {
```

Comment:
I think this should be `BaseFacetUi` https://blogs.oracle.com/tor/entry/code_advice_3_don_t